assets/src/stories-editor/plugins/story-settings-panel.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { metaToAttributeNames } from '../helpers';
+
+/**
+ * WordPress dependencies
+ */
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+import { compose } from '@wordpress/compose';
+
+import {
+	withSelect,
+	withDispatch,
+} from '@wordpress/data';
+
+import {
+	SelectControl,
+	RangeControl,
+} from '@wordpress/components';
+
+let MetaFields = ( props ) => {
+	const {
+		meta,
+		updateMeta,
+	} = props;
+
+	const {
+		autoAdvanceAfter,
+		autoAdvanceAfterDuration,
+	} = metaToAttributeNames( meta );
+
+	const autoAdvanceAfterOptions = [
+		{ value: '', label: __( 'Manual', 'amp' ) },
+		{ value: 'auto', label: __( 'Automatic', 'amp' ) },
+		{ value: 'time', label: __( 'After a certain time', 'amp' ) },
+		{ value: 'media', label: __( 'After media has played', 'amp' ) },
+	];
+
+	let autoAdvanceAfterHelp;
+
+	if ( 'media' === autoAdvanceAfter ) {
+		autoAdvanceAfterHelp = __( 'Based on the first media block encountered on the page', 'amp' );
+	} else if ( 'auto' === autoAdvanceAfter ) {
+		autoAdvanceAfterHelp = __( 'Based on the duration of all animated blocks on the page', 'amp' );
+	}
+
+	return (
+		<>
+			<SelectControl
```

Comment:
Can we add an explanation that this changes the setting globally for the whole story? I don't think it's clear otherwise.